pkg/file/splitter/internal/job.go

```diff
@@ -47,14 +48,15 @@ type SimpleSplitterJob struct {
 	cursors    []int    // section write position, indexed per level
 	hasher     bmt.Hash // underlying hasher used for hashing the tree
 	buffer     []byte   // keeps data and hashes, indexed by cursors
-	toEncrypt  bool     // to encryrpt the chunks or not
+	tagg       *tags.Tag
```

Comment:
i hope this is a typo,

Reply:
nope..
t - is testing
tag- is already there 
tags is already there 
tagger - should be a interface
so i chose tagg :-)

Reply:
oh. noo